incapsula/resource_acl_security_rule.go

```diff
@@ -166,6 +167,26 @@ func resourceACLSecurityRuleUpdate(d *schema.ResourceData, m interface{}) error
 	return resourceACLSecurityRuleCreate(d, m)
 }
 
+func testAccStateACLSecurityRuleID(s *terraform.State) (string, error) {
```

Comment:
Why is a `test` func in this resource file? It's not called explicitly from anywhere else in this file. Is it called implicitly by something else?